src/django/api/tests.py

```diff
@@ -69,6 +69,17 @@ def setUpTestData(cls):
             created_by=cls.contributor,
         )
 
+        draft = Submission.objects.create(
+            boundary=cls.boundary_2,
+            shape=RALEIGH_FAKE_ZIGZAG,
+            created_by=cls.contributor,
+        )
+
+        # Created_at auto-populates so need to update to earlier date
+        Submission.objects.filter(pk=draft.pk).update(
+            created_at=datetime(2022, 10, 2, hour=9, tzinfo=timezone.utc)
+        )
```

Comment:
At your option, as `update` feels more of a bulk-operation thing.
```suggestion
        draft.created_at = datetime(2022, 10, 2, hour=9, tzinfo=timezone.utc)
        draft.save()
```

Reply:
At some future point we might create a datetime helper that could take just a number of hours after 10/1/2022 and produce the wanted datetimes.

Reply:
We're also specifying a value for `created_at` in the creation of all other instances here. Do they also not work correctly? Or can we just use do that?

Reply:
Perhaps something like this:

```python
def dt(date: str, hour: int = 0) -> datetime:
    """
    Turns a string of type "YYYY-MM-DD" into a datetime.
    Uses the optional hour value if specified, else defaults to 0.
    """
    year = int(date[:4])
    month = int(date[5:7])
    day = int(date[8:])

    return datetime(year, month, day, hour, tzinfo=timezone.utc)
```
```python
   created_at=dt('2022-10-03', hour=15),
```

Reply:
> We're also specifying a value for created_at in the creation of all other instances here. Do they also not work correctly? Or can we just use do that?

I did it that way to reduce the variation between test runs. If it became important to structure test cases so that action B took place in between actions A and C, I didn't want to rely on the microsecond timing between parts of this test method executing.

I might be missing your point, though!

Reply:
> I might be missing your point, though!

My point is specifying `created_at` in

```python
Submission.objects.create(created_at=...)
```

vs changing the value after creation:

```python
s = Submission.objects.create(...)
s.created_at = ...
s.save()
```

Does that make sense?

Reply:
Ah, yes it does!

Reply:
So should we change it everywhere then? Not just here?

Reply:
Alternatively, if we just created the items in the right order, that should be enough, and we may drop `created_at=` specification completely.

Reply:
Ah, good call. The latter option seems like it would be more intuitive/readable, so I'm for that! However I understand Jacob's point on reducing potential variations. Maybe a compromise is dropping the specification to improve readability and adding a test to check that the order of execution of submissions creation is correct?

Reply:
I retract my concern :-)

For now I bet a comment is fine, we can always add the test later.

Reply:
Thanks all, will do a comment! If there's time an activity log test could potentially be dual-purpose and confirm serializer output as well as the correct order, too.